src/historic-quote/historic-quote.service.ts

```diff
@@ -276,28 +298,59 @@ export class HistoricQuoteService implements OnModuleInit {
     const startPaddedQ = moment.unix(start).utc().startOf('day').subtract('1', 'day').toISOString();
     let endQ: any = moment.unix(end).utc().endOf('day');
     endQ = endQ.isAfter(today) ? today.toISOString() : endQ.toISOString();
-    const addresses = _addresses.map((a) => a.toLowerCase());
 
-    const addressesString = addresses.map((a) => `'${a}'`).join(', ');
+    const enabledProviders = this.priceProviders[blockchainType]
+      .filter((p) => p.enabled)
+      .sort((a, b) => a.priority - b.priority)
+      .map((p) => p.name);
```

Comment:
you can do the `join` here and be over with it

Reply:
done